api/persistence.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"encoding/json"
+	"syscall"
+)
+
+
+type Persistence interface {
+	// [new and lock the filelock and] store the just created sandbox
+	NewSandbox(*Sandbox) error
+	// NewContainer(*Container) error
+	// [lock the filelock and] build sandbox from existing running sandbox
+	GetSandbox(id string, SandboxBuilder) (*Sandbox, error)
```

Comment:
Maybe just `Sandbox(...)` (https://golang.org/doc/effective_go.html#Getters).

Reply:
the 'get' is from 'get/put' pair. It is not a pure "read" operation, it might lock the filelock or get a reference.